client/lib/cart-values/cart-items.js

```diff
@@ -319,12 +319,14 @@ function businessPlan( slug, properties ) {
  *
  * @param {Object} productSlug - the unique string that identifies the product
  * @param {string} domain - domain name
+ * @param {Number} volume
```

Comment:
It seems more natural to me to use the word `quantity` to specify how many of a specific product item we want in the cart. Maybe we can have some others chime in. `volume` sounds a bit odd to me and maybe misleading (though maybe I don't understand this well enough!)